index.html

```diff
@@ -0,0 +1,874 @@
+        <span class="navbar-toggler-icon"></span>
+        </button>
+      </div>
+      <!-- White part of navbar -->
+      <div class="font-weight-bold collapse navbar-collapse align-self-stretch bg-white" id="navbarSupportedContent">
+        <form class="form-inline my-2 my-lg-0 input-group d-flex d-lg-none">
+          <input class="form-control mr-sm-2 input-group-text" type="search" placeholder="Search" aria-label="Search">
+          <button class="btn my-2 my-sm-0 input-group-append" type="submit"><i class="material-icons">search</i></button>
+        </form>
+        <ul class="d-flex align-items-end navbar-nav ml-lg-auto mr-auto mr-md-0 ml-md-6">
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">U.S. <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">World <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Business <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Tech & Science <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Culture <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Newsgeek <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Sports <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Health <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">The Debate <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Vantage <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Weather <span class="short"> > </span></a></h6>
+              </li>
+        </ul>
```

Comment:
- [ ] The navbar menus are not displaying like the original page. You should check the width of the `<ul>` element and make it look like the original page.